common/src/main/java/com/wynntils/gui/render/buffered/BufferedFontRenderer.java

```diff
@@ -355,4 +357,110 @@ public void renderText(
                     shadow);
         }
     }
+
+    public void renderTextsWithAlignment(
+            PoseStack poseStack,
+            MultiBufferSource.BufferSource bufferSource,
+            float x,
+            float y,
+            List<TextRenderTask> toRender,
+            float width,
+            float height,
+            HorizontalAlignment horizontalAlignment,
+            VerticalAlignment verticalAlignment) {
+        float renderX =
```

Comment:
Come to think of it, that remark probably goes for this as well. Seems this exact code is already copied twice in FontRenderer...

And the same for renderY below.

Reply:
You can skip this too, then.